contrib/reformat-code.py

```diff
@@ -11,31 +11,36 @@
 import subprocess
 
 CLANG_FORMATTERS = [
+    "clang-format-11",
     "clang-format-13",
     "clang-format",
 ]
+CLANG_DIFF_FORMATTERS = [
+    "clang-format-diff-11",
```

Comment:
Do you actually want to add 11 or not? I thought there was features needed from 13.

Reply:
On https://github.com/fwupd/fwupd/pull/3526 we removed the align consecutive declarations that required 13 as it not only affects structs but all declarations